### PR TITLE
Zigbee gateway (cid) support

### PR DESCRIPTION
**Pull request by imbenwolf**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -69,10 +70,11 @@ program
 	.option('--ip <ip_addr>', 'ip address of device')
 	.option('--id <id>', 'id of device')
 	.option('--key [key]', 'key of device')
+	.option('--cid [cid_zigbee_device]', 'if specified, use device id of zigbee gateway and cid of subdevice to set its property')
 	.option('--set <set>', 'value to set')
 	.option('--raw-value', 'pass the raw set value without attempting to parse it from a string')
 	.option('--dps [dps]', 'DPS index to set', 1)
-	.option('--protocol-version [version]', 'tuya protocol version', parseFloat, 3.1)
```
Comment (codetheweb) [inline cli.js:75]: This is breaking, probably should leave it as 3.1 for now.

**Review by codetheweb — CHANGES_REQUESTED** [review]
Looks pretty good, thanks!

File: cli.js (hunk 1 above)
Comment (imbenwolf) [reply cli.js]: Done!

**Review by codetheweb — APPROVED** [review]

**Comment by codetheweb** [comment]
Published in v1.16.0

**Pull request merged**